src/Interfaces/stochasticstyles.jl

```diff
@@ -119,15 +113,21 @@ length. These will be reported as columns in the `DataFrame` returned by [`lomc!
 step_stats(v) = step_stats(StochasticStyle(v))
 
 """
-    fciqmc_col!(w, ham, add, num, shift, dτ)
-    fciqmc_col!(::StochasticStyle, args...)
+    apply_column!(v, O, i, x, boost=1) -> stats::Tuple
```

Comment:
```suggestion
    apply_column!(v, op, addr, num, boost=1) -> stats::Tuple
```
While single letter names are clearly more elegant, I think I prefer to old way of naming the arguments as it was more suggestive of meaning.